### DISCUSSION TITLE
Frage zu den Pylontech US5000 Spannungen

### DISCUSSION THREAD

**Discussion by swingstate** [hoylabs/OpenDTU-OnBattery#817] [General]
Ich habe meine Anlage nun auf 2x US5000 in Parallelschaltung umgebaut. Das lief problemlos soweit und die SoC Daten kommen via CAN Bus über eines der mir entworfenen PCBs an. Ich habe nun ins Wiki geschaut und der Punkt bzgl. der Pylontech-Spannungen ist mir nicht ganz klar. 

Meinen 150/35 habe ich wie folgt eingestellt: 

Batterie: 48V
Max. Ladestrom 35A
Absorptionsspanung: 52,4V
Erhaltungsspannung: 52V

Den DPL habe ich wie folgt eingestellt:

Akku SoC - Start: 50%
Akku SoC - Stop: 19%
Start Solar passthrough: 97%

DC Spannung - Start: 50V
DC Spannung - Stop: 48V
Start Solar-passthrough: 52,2
Stop Solar-passthrough: 51,8
Lastkorrekturfaktor: 0,0015 (unverändert von meinen PowerQueen Batterien übernommen).

Habt ihr dazu vielleicht Vergleichswerte? 

Mir kommen meine Spannungen etwas niedrig vor. Ich werde dann das Wiki bzgl. Pylontech entsprechend um diese Bespielkonfiguration erweitern, so dass die Spannungseinstellungen auch für andere eindeutiger sind. 

Meinen ursprünglichen Post werde ich ebenfalls aktualisieren bzw. eine zweite Beispielkonfiguration dokumentieren.

Frohe Ostern!

**Comment by genmad** [comment]
Hallo swingstate,

laut https://www.victronenergy.com/live/battery_compatibility:pylontech_phantom
 Empfiehlt Victron 52V absorption und 51V float.
Diese Einstellungen hatten bei mir bis 98% SOC geladen und dann im float Modus dafür gesorgt, dass meine Batterie ( Us5000c an RS450/100 mit ca. 4,4 kWp Solar) im float Modus immer leicht entladen wurde. 
Bin gerade auch am rumexperimentieren welche Einstellungen mir gefallen könnten. Habe die Spannungen heute auf 52,5V und 51,5V angehoben. Mal sehen ob mir das besser gefällt.

Auf Grund meiner hohen Erzeugungsleistung und dem Wunsch der Batterie ab 90%Soc nur noch mit 20A geladen zu werden habe ich den Solarpassthrough bei 90% gesetzt. Soc start steht bei mir bei 20% und Soc stop bei 10%.

**Reply by genmad** [reply]
Bin wieder auf die empfohlenen 52V und 51V zurückgegangen, da man ja nicht immer bis 100% laden soll, da dies die Batterie in Mitleidenschaft ziehen kann. 52V Absorptionsspannung führen bei mir zu 98%Soc.

Nun habe ich aber die Frage, wann bzw. Wie oft man auf 100% laden lassen soll, da ja das BMS über Ladungszählung seinen Soc bestimmt und wenn man nie bis 100% lädt der Soc Wert des BMS weglaufen kann und sich das bms dann auch nie rekalibrieren kann.

So weit habe ich das mal gelesen. Wie habt ihr das gelöst? BMS Kalibrierung vs. Batterie haltbarkeit?

**Reply by knopers1** [reply]
was machst du für ein Drama daraus? Du hast 5 Jahre Garantie bekommen, und das Teil kostet gerade Mal 1200€... 
Ich lade den Akku mit 52.8V voll, habe aber SOC Start auf 80% stehen... Je nachdem wie es auskommt, wird der Akku zu 100% aufgeladen, manchmal aber nur knapp über die 80% Grenze. Es ist halt vom Wetter oder eher von den Sonnentunden am Tag abhängig.... Stell das Teil ein, und lass es stehen. Es wird sich schon selbst wunderbar regeln....

**Reply by swingstate** [reply]
Ich finde die Frage relevant, was die Haltbarkeit über die 5 Jahre hinaus angeht. @schlimmchen, ihr habt ja in den vergangenen Tagen einiges am Ve.Direct Protokoll gewerkelt - Vielleicht weist Du, ob sich die Werte des MPPTs für Absorption und Erhaltung via Ve.Direct TX ändern lassen? Dann könnte man das ggfs. automatisieren und zwischen 52V/51V und höheren Spannungen automatisiert wechseln. Quasi verschiede Ladeprofile.

**Reply by schlimmchen** [reply]
Ja, das geht (per HEX-Protokoll).

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/3578416/423c8e99-3f8d-40ec-9a14-f4874a3e1b5d)

Daher kein Go für eine Unterstützung in OpenDTU-OnBattery, meiner Meinung nach.

Ich empfehle ehrlich gesagt den SoC zu ignorieren und rein basierend auf den Spannungen den DPL und den Laderegler einzustellen. Also immer auf nur 80% oder 90% zu laden (passende Spannung im Laderegler einstellen), egal was der SoC sagt und wie er abdriftet.

**Reply by swingstate** [reply]
Die Warnung bezieht sich auf den NVM des MPPT? Dann ist das keine Option.

**Reply by schlimmchen** [reply]
Hehe, ich dachte mir schon in der Vorschau des Kommentars, dass das eigentlich zu wenig Kontext ist :grin:

Ja, das ist aus der VE.Direct HEX Beschreibung und steht über der Tabelle mit den hier diskutierten Einstellungen:

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/3578416/818e325b-4016-4857-ae3c-0f71c31d8f7a)

> Dann ist das keine Option.

Das sehe ich auch so.

**Reply by swingstate** [reply]
Meine Spannungen sehen nach etwas Recherche und Rechnen nun so aus: 

MPPT:
Absorptionsspanung: 52,4V (3,49V / Zelle)
Erhaltungsspannung: 51V (3,4V / Zelle)

Den DPL habe ich wie folgt eingestellt:

Akku SoC - Start: 50%
Akku SoC - Stop: 19%
Start Solar passthrough: 90%

DC Spannung - Start: 50V

Folgende Werte muss ich noch weiter testen.
DC Spannung - Stop: 48V
Start Solar-passthrough: 52,2
Stop Solar-passthrough: 51
Lastkorrekturfaktor: Muss ich auch noch ran

Bisher ist mein subjektiver Eindruck das die Batterien weitaus stabiler sind was die Spannungen angeht als die zwei Powerqueen.

**Reply by genmad** [reply]
@schlimmchen Falls ich dich richtig verstehe ist das dann eine indirekte Empfehlung die Pylontech nicht über can bus an ODoB anzubinden? 

Warum verstehe ich das so? 
Du sagst man solle den Soc ignorieren und nur über die Spannung zu gehen. Soweit ich weiß kann aber der soc nur ignoriert werden, wenn keine Kommunikation mit der Batterie stattfindet. Im DpL wird ja alles über Soc geregelt und nur wenn der nicht kommt wird über Spannung geregelt. Soc start kann man ja soweit runtersetzen wie man will, so dass mir nur Soc Stop Kopfschmerzen macht. Wenn der Soc wegläuft, dann habe ich ja nur die Chance das über keine Batterie Verbindung hinzubekommen, oder?

Habe ich da richtig interpretiert?

**Reply by schlimmchen** [reply]
> Soweit ich weiß kann aber der soc nur ignoriert werden, wenn keine Kommunikation mit der Batterie stattfindet.

Nope.

![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/3578416/86edb6a0-6bee-4133-ac22-ae75a6180a2b)

Seit dem vorletzten Release (glaube ich, nagel mich nicht fest).

**Comment by swingstate** [comment]
Ich konnte die 2 Batterien heute das erste Mal vollladen. Mit 52,40V Absorptionsspannung lädt die Batterie bis 96%, bleibt dort knapp 2h bis auf die 51V Float gewechselt wird. Werde nun mal 52,8V testen - macht dann 3,52V pro Zelle. 


bzgl. meiner Anlage - Habe heute auch noch 2 baugleiche 385W Panel dazu installiert und konnte bei leichter Bewölkung mit 1,85 KW (Abzüglich 150W Grundverbrauch) mit bis zu 33A laden. Der MPPT wurde dabei bis 60° warm - muss noch ein Lüfter dran.

**Reply by genmad** [reply]
Ich habe mich gewundert, dass du fast 10kWh Speicher an deine Anlage hängst. Da habe ich mich gefragt was machst du denn mit dem ganzen Strom? Du musst ja fast 6h laden um die leeren Batterien am Tag wieder voll zu bekommen.
Ich habe, wie schon geschrieben, 5kWh Speicher und auch 150W Grundlast. Ich komme mit meiner Batterie gut über die Nacht.

Frage nur aus Interesse an den dahinterstehenden Amortisationszeitraum.

**Reply by swingstate** [reply]
Neben der Grundlast liegt unser täglicher Verbrauch zwischen 8 - 15Kwh, meist über 11Kwh. Der Verbrauch erfolgt vor allem dann, wenn die Sonne weniger scheint (morgens, und vor allem Abends). Das war vor einem Jahr noch anders, aber Dinge ändern sich mit der Zeit. Ich habe nun 2,3Kwp installiert. Mehr wird's nicht werden. Ggfs. tausche ich den MPPT noch gegen den 150/45 da der 35er hier limitiert. Bin auch noch am überlegen ob ich die Batterie nur bis 85 oder 90% laden um viele Jahre was von ihnen zu haben.

**Reply by swingstate** [reply]
Mit 52,8V geht es bis 98%. Selbst wenn ich die Ladespannung dann auf max. 53,25V (3,55V pro Zelle) einstelle tut sich da nichts mehr. Ich belasse es daher bei maximal 52,8V.
